src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -175,6 +222,76 @@ export default class MissionWindow extends Component<ThemeProps, State> {
       endMissionIndex: value[1],
     });
   }
+  // GeoFence BoundingBox
+
+  private onChange(event: React.ChangeEvent<HTMLInputElement>): void {
+    const { checklist } = this.state;
+    if (!(event.target.name in checklist)) return;
+
+    const name = event.target.name as GeofenceChecklistType;
+    const value = parseInt(event.target.value, 10) || 0;
+    switch (name) {
+      case 'geofenceTop':
+        ipc.postUpdateBoundingBoxes(true, {
+          name: 'Bounding Box',
```

Comment:
im pretty sure the name wouldnt be Bounding Box, probably should be Geofencing or Geofencing Box

make sure to update the other three cases too and change Bounding Box to your geofencing name